src/components/CommunityEvents/index.tsx

```diff
@@ -0,0 +1,221 @@
+//Libraries
+import React from "react"
+import { useI18next, useTranslation } from "gatsby-plugin-react-i18next"
+import {
+  Box,
+  Center,
+  Divider,
+  Flex,
+  Grid,
+  GridItem,
+  Heading,
+  Icon,
+  Text,
+} from "@chakra-ui/react"
+import { FaDiscord } from "react-icons/fa"
+import { DateTime } from "luxon"
+
+// Components
+import ButtonLink from "../ButtonLink"
+import Link from "../Link"
+import Translation from "../Translation"
+
+// Utils
+import { trackCustomEvent } from "../../utils/matomo"
+
+// Hooks
+import { useCommunityEvents } from "./useCommunityEvents"
+
+const matomoEvent = (buttonType) => {
+  trackCustomEvent({
+    eventCategory: "CommunityEventsWidget",
+    eventAction: "clicked",
+    eventName: buttonType,
+  })
+}
+
+const renderEventDateTime = (
+  date,
+  language,
+  params = {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+    hour12: false,
+    hour: "numeric",
+    minute: "numeric",
+  }
+) => {
+  return DateTime.fromISO(date).setLocale(language).toLocaleString(params)
+}
+
+const EventLink = (props) => <Link fontWeight="700" {...props} />
+
+const Event = ({ event, language, type }) => {
+  const { date, title, calendarLink } = event
+  const params = {
+    year: "numeric",
+    month: "short",
+    day: "numeric",
+  }
+
+  return (
+    <Grid gap={6} templateColumns="100px auto">
```

Comment:
In terms of UI, this is how I see it now:
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/8e6f5ebd-f593-419c-af75-53f18585b45b)

I think we should force the datetime column to always be a one line text.

```suggestion
    <Grid gap={6} templateColumns="auto 1fr">
```